tests/unit/test_handler.py

```diff
@@ -0,0 +1,18 @@
+import json
```

Comment:
in pytest test files have to be named `test_` else it will give error that no tests found.

Reply:
ok